box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/CollaborationsFragment.java

```diff
@@ -101,14 +102,12 @@ public void onSaveInstanceState(Bundle outState) {
     public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
         CollaboratorsAdapter.ViewHolder holder = (CollaboratorsAdapter.ViewHolder) view.getTag();
         if (holder != null && holder.collaboration != null) {
-            ArrayList<BoxCollaboration.Role> rolesArr = getRoles();
-
             BoxCollaborator collaborator = holder.collaboration.getAccessibleBy();
 
             String userId = mController.getCurrentUserId();
             boolean isCollaboratorCurrentUser = collaborator != null && collaborator.getId().equals(userId);
 
-            if ((rolesArr == null || rolesArr.size() == 0) && !isCollaboratorCurrentUser) {
+            if (rolesArr == null || (rolesArr.size() == 0 && !isCollaboratorCurrentUser)) {
```

Comment:
thought we discussed having a toast for the rolesArr being null.

Reply:
`Unable to fetch` dialog is for null case.. disabling collaborators in the list in case of empty rolesArr is handled by myo in the new UI